### ISSUE TITLE
CKEditor: Uploading an image hides the entire upload/select image widget

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#3482]
**Describe the bug**
This seems to be a regression with 1.12, and could be related to #3355 

**Steps To Reproduce**
To reproduce the behavior:
1. Create a page or post.
2. Click on the "Image" toolbar button on the WYSIWYG
3. Switch to the "Upload an image" tab.
4. Click on the "Choose file" button and upload an image...

**Actual behavior**
The entire upload/select image widget disappears:

<img width="522" alt="screen shot 2019-01-16 at 3 43 45 pm" src="https://user-images.githubusercontent.com/2423362/51252875-9f095d00-19a5-11e9-8116-7c8d4a229138.png">

**Expected behavior**
The image gets uploaded as per usual.

**Comment by klonos** [comment]
...clicking on "Insert" inserts the image in the editor as expected though.

**Comment by klonos** [comment]
While testing this, I have tried uploading an image several times (because the image widget was disappearing, I thought that the upload failed). Once the image was inserted, I double-clicked on it in the WYSIWYG to edit it...

- The "Edit image" dialog was switched to the "Select from library" tab 👎 (I'd expect the "Upload an image" tab to be selected). At this point, the image library was not expanded, so I assumed that the image was not successfully uploaded nor added to the library.

- I switched to the "Upload an image" tab. The image widget was shown, with a "remove" button available next to the image name, as expected 👍 

- Switching back to the "Select from library" tab expanded the image library to reveal multiple copies of the same image, from the multiple attempts to upload it (because the widget was disappearing)
